lib/helper/timepass.php

```diff
@@ -616,4 +616,51 @@ public static function getTimePassesCount() {
 
 		return $model->getTimePassesCount();
 	}
+
+	/**
+	 * Has user active timepass or not
+	 *
+	 * @return bool
+	 */
+	public static function userHasActiveTimepass() {
+
+		$timepasses = self::getAllTimePasses();
+		if (!count($timepasses)) {
+			return FALSE;
+		}
+
+		$timepassesIds = array();
+		foreach ($timepasses as $timepass) {
+			$timepassesIds = self::getTokenizedTimePassId($timepass['pass_id']);
+		}
+
+		$clientOptions = tx_laterpay_helper_config::getPhpClientOptions();
+		$laterpayClient = new LaterPay_Client($clientOptions['cp_key'], $clientOptions['api_key'], $clientOptions['api_root'],
+			$clientOptions['web_root'], $clientOptions['token_name']);
+
+		$result = $laterpayClient->get_access($timepassesIds);
+
+		$articles = $result['articles'];
+		foreach ($articles as $article) {
+			if ($article['access']) {
+				return TRUE;
+			}
+		}
+		return FALSE;
+	}
+
+	/**
+	 * Has user active timepass or not
```

Comment:
Given the description of userHasActiveTimepass, this description is obviously wrong.